django/contrib/admin/static/admin/css/base.css

```diff
@@ -518,7 +518,7 @@ select[multiple] {
     padding: 10px 15px;
     border: none;
     border-radius: 4px;
-    color: var(--button-fg);
+    color: var(--body-fg);
```

Comment:
This unfortunately doesn’t seem clear enough to me. It technically passes WCAG contrast checks but is still in my opinion too low contrast for such a critical component, and also as-is makes those buttons more prominent than primary form buttons.